### PR TITLE
not all changes change the directory structure though

### PR DESCRIPTION
**Pull request by cordoval**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,9 +17,9 @@ $ git clone git://github.com/Ocramius/Ocramius.git
 You will then need to run a composer installation:
 
 ```sh
-$ cd ProxyManager
+$ cd Ocramius
```
Comment (cordoval) [inline CONTRIBUTING.md:20]: this was a genuine fix

Comment (cordoval) [inline CONTRIBUTING.md:20]: @Ocramius ^^ but I see you are not very collaborative so I will just watch now, thought you were different :blush:

Comment (Ocramius) [inline CONTRIBUTING.md:20]: Yup, but a load of other changes were wrong/unrelated

Comment (cordoval) [inline CONTRIBUTING.md:20]: so you grilled the right with the unright ... :blush:

**Comment by cordoval** [comment]
@Ocramius i have a question, so this repo is a console to report bugs into gists? or issue tickets? wonder if I should create a Gush command to do this.

**Comment by cordoval** [comment]
closing issues as I guess i will just watch and learn in silence :blush:

**Comment by Ocramius** [comment]
> @Ocramius i have a question, so this repo is a console to report bugs into gists? or issue tickets? wonder if I should create a Gush command to do this.

I actually started it as a joke [started](https://twitter.com/rosstuck/status/470263269232750592) by @rosstuck

I then decided to convert it into something semi-useful, which is basically what I do into IRC, but as a command. 

I'll likely plug this into an IRC bot at some point.

**Comment by rosstuck** [comment]
To be fair, @macnibblet was the real creator: https://github.com/Roave/RoaveTrack/issues/2#issuecomment-44187422

**Pull request closed without merging**